src/mapblock.cpp

```diff
@@ -66,14 +66,13 @@ static const char *modified_reason_strings[] = {
 	MapBlock
 */
 
-MapBlock::MapBlock(Map *parent, v3s16 pos, IGameDef *gamedef, bool dummy):
+MapBlock::MapBlock(Map *parent, v3s16 pos, IGameDef *gamedef):
```

Comment:
Looks like there wasn't a single caller passing the dummy value.
Good riddance!